internal/monitors/pyrunner/linux.go

```diff
@@ -21,7 +23,11 @@ func procAttrs() *syscall.SysProcAttr {
 }
 
 func pythonBinaryExecutable() string {
-	return filepath.Join(os.Getenv(constants.BundleDirEnvVar), "lib64", "ld-linux-x86-64.so.2")
+	if strings.Compare(runtime.GOARCH, "arm64") == 0 {
+		return filepath.Join(os.Getenv(constants.BundleDirEnvVar), "lib", "ld-linux-aarch64.so.1")
+	} else {
```

Comment:
From golint:

`if block ends with a return statement, so drop this else and outdent its block`

Reply:
done